ibm/service/vpc/data_source_ibm_is_instance.go

```diff
@@ -59,7 +59,32 @@ func DataSourceIBMISInstance() *schema.Resource {
 				Computed:    true,
 				Description: "Indicates whether the metadata service endpoint is available to the virtual server instance",
 			},
+			isInstanceMetadataService: {
```

Comment:
As of now the values are not getting populated in any of the data sources, Please review the changes again